src/Symfony/Component/Validator/Tests/Constraints/FileValidatorTest.php

```diff
@@ -555,7 +555,7 @@ private function validExtensionProvider(): iterable
     {
         yield ['test.gif'];
         yield ['test.png.gif'];
-        yield ['ccc.txt'];
+        yield ['ccc.TXT'];
```

Comment:
Thanks. You actually need to create the test fixture file `ccc.TXT`.

Reply:
And use it (instead of replacing ccc.txt).

Reply:
okay, did so.